lab-bundle-spec.md

```diff
@@ -271,13 +269,92 @@ No additional attributes
   id: primary_domain
 ```
 
-##### Future Resource Types
+##### AWS Account (aws_account)
+
+attribute   | required | type       | notes
+----------- | -------- | ---------- | ----------------------------------------
+default_policy.allow_aws_dedicated_instances |          | boolean | Default false.
+default_policy.allow_aws_ondemand_instances  |          | array   | Array of [EC2 instance types](#valid-eC2-instance-types) that are valid Ondemand usage. Default none.
+default_policy.allow_aws_rds_instances       |          | array   | Array of [EC2 instance types](#valid-eC2-instance-types) that are valid for RDS usage. Default none.
+default_policy.allow_aws_spot_instances      |          | boolean | Default false.
+default_policy.allow_aws_subnet_deletion     |          | boolean | Default false.
+default_policy.allow_aws_vpc_deletion        |          | boolean | Default false.
+startup_script.type              |          | string  | The type of startup script. Only `cloud_formation` is supported.
+startup_script.path              |          | path    | Relative path to a directory tree with the script contents.
+startup_script.custom_properties |          | array   | Array of pairs. See [Custom Script Properties](#custom-script-properties) for details.
+cleanup_script.type              |          | string  | The type of cleanup script. Only `cloud_formation` is supported.
+cleanup_script.path              |          | path    | Relative path to a directory tree with the script contents.
+cleanup_script.custom_properties |          | array   | Array of pairs. See [Custom Script Properties](#custom-script-properties) for details.
+user_policy                      |          | path    | Relative path to a [JSON policy](https://awspolicygen.s3.amazonaws.com/policygen.html) document
+variant                          |          | enum*   | Specify a Qwiklabs fleet name.
+
+```yml
+- type: aws_account
+  id: the_account
+  variant: sts
+  startup_script:
+    type: cloud_formation
+    path: ./lab.template
+    custom_properties:
+      - key: userNameWindows
+        value: student
+  user_policy: ./student.policy
+  default_policy:
+    allow_aws_dedicated_instances: false
+    allow_aws_ondemand_instances: false
+    allow_aws_rds_instances: ['db.t2.micro']
+    allow_aws_spot_instances: false
+    allow_aws_subnet_deletion: false
+    allow_aws_vpc_deletion: false
+```
+
+###### Valid Variants for AWS Account
```

Comment:
please make this more consistent with the `Variants` section in GCP Project

Reply:
Do you mean just make the title "Variants"?

Reply:
yes